### ISSUE TITLE
Query Samples across different Sample Templates

### ISSUE THREAD

**Issue by rabuono** [seek4science/seek#1048]
Very similar to issue #595, but using Templates instead of Sample Types to perform the search.
The question goes from:
"As a User I would like to search for Samples in Sample Type 5 that are from Species Solanum tuberosum (Sample Type 1)"
To:
"As a User I would like to search for Samples derived from Template 5 that are from Species Solanum tuberosum (derived from Template 1)"
All Sample Types originated from Template 5 and Template 1 would be covered in the search.

**Comment by rabuono** [comment]
It might be best to change the text in 1 to:
`Select the Template used to create the parent Samples from which your Samples of interest derive from.`

And in 2 to:
`Select the Template used to create the child Samples that derive from your Samples of interest.`


![Image](https://user-images.githubusercontent.com/77321541/179233035-0bac4732-65b3-44e6-85a9-01799020afc0.png)

**Comment by rabuono** [comment]
- [ ] It would be best if the result table also contained name and SEEK ID of the Project and Study/assay to which the resulting Samples are linked to.

**Comment by rabuono** [comment]
Correct, these are minor adjustments only. The original ones where for **directly** derived samples. This is not the case for the feature anymore.
Also, they had `samples` with lowercase `s`. We generally use `Samples` when referring to SEEK Samples.

**Comment by rabuono** [comment]
- [ ] Remove `out of` from the `X samples visible to you out of Y` result message to avoid disclosing Sample value information.

**Comment by floradanna** [comment]
- [ ] Improve the GUI as in the screenshot below
  - The "Advanced filtering" rectangle should be collapsible
  - The current "Query option" rectangle is not needed


<img width="704" alt="Screenshot 2022-07-18 at 11 16 42" src="https://user-images.githubusercontent.com/51486716/179481120-4a5ed22e-2ad0-4403-be5a-9a068de6bc9f.png">

**Comment by vdkkia** [comment]
@floradanna All the changes except the arrangement of the boxes for Input/Output have been made.
I think if we change the arrangement of the text boxes, it will be in contrast with the upper section (The Template drop-down lists and text box)

**Comment by floradanna** [comment]
I think this issue is solved and it can be closed
